pkgs/by-name/cg/cgterm/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, SDL
+}:
+
+stdenv.mkDerivation {
+  pname = "cg-term";
+  version = "unstable-2016-02-25";
```

Comment:
Looks like it's [1.7b2.](https://github.com/MagerValp/CGTerm/commit/01e35d64c29bccee52211b0afc66035a10e4792a)